lib/word.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'node'
+require 'byebug'
+
+class Word
+  def initialize(word = nil)
+    @word = word
+  end
+
+  def add
+    array_chars = @word.split('')
+    Node.new.add(array_chars, $prefix_tree)
+  end
+
+  def include?
+    array_chars = @word.split('')
+    Node.new.nodes_present?(array_chars)
```

Comment:
Use class methods fo purposes like that
```
class Node
  def self.nodes_present?(array_chars)
end
####
Node.nodes_present?
```

This should not be implemented, this just an example, because Node class should be free of any logic, it is just information chain